src/firecfg/firecfg.config

```diff
@@ -698,6 +698,8 @@ opera
 opera-beta
 opera-developer
 orage
+#org.gnome.gedit # text editor (see #6002)
+org.gnome.seahorse.Application
```

Comment:
Does this work w/o redirect profile?

Reply:
> Does this work w/o redirect profile?

Added.

Reply:
The other redirects for this bug use just the lastname approach: https://github.com/netblue30/firejail/blob/master/etc/profile-m-z/Maps.profile

Reply:
> The other redirects for this bug use just the lastname approach:
> https://github.com/netblue30/firejail/blob/master/etc/profile-m-z/Maps.profile

firecfg recognizes the redirect profile with the following PR:

* #6674